package/MDAnalysis/guesser/DefaultGuesser.py

```diff
@@ -0,0 +1,155 @@
+from .base import GuesserBase
+import numpy as np
+import warnings
+
+import re
+
+from ..lib import distances
+from . import tables
+
+
+class DefaultGuesser(GuesserBase):
+    context = 'default'
+ 
+    def __init__(self, atoms):
+        super().__init__(atoms)
+        self._guess = {'mass': self.guess_masses,
+                       'type': self.guess_types}
+        self._rank = {'mass': 2,
+                      'type': 1
+                      }
+
+    def guess_masses(self):
+        """Guess the mass of many atoms based upon their type
+
+        Returns
+        -------
+        atom_masses : np.ndarray dtype float64
+        """
+        if hasattr(self._atoms, 'elements'):
+            atom_types = self._atoms.elements
+        else:
+            atom_types = self._atoms.types
+
+        self.validate_atom_types(atom_types)
+        masses = np.array([self.get_atom_mass(atom_t)
+                           for atom_t in atom_types], dtype=np.float64)
+        return masses
+
+    def validate_atom_types(self, atom_types):
+        """Vaildates the atom types based on whether they are available
+        in our tables
+
+        Parameters
+        ----------
+        atom_types
+          Type of each atom
+
+        Returns
```

Comment:
Don’t show return none. Add a”Raises” section. See the User Guide on writing documentation and the example for “numpy docs”.